src/SFA.DAS.ApprenticePortal.UnitTests/FeaturesSteps/HomePageSteps.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Net;
+using System.Threading.Tasks;
+using AutoFixture;
+using FluentAssertions;
+using SFA.DAS.ApprenticePortal.Web.Pages;
+using SFA.DAS.ApprenticePortal.Web.Services.OuterApi;
+using TechTalk.SpecFlow;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+
+namespace SFA.DAS.ApprenticePortal.UnitTests.FeaturesSteps
+{
+    [Binding]
+    [Scope(Feature = "HomePage")]
+    public class HomePageSteps : StepsBase
+    {
+        private TestContext _context;
+        private Guid _apprenticeId;
+        private Apprenticeship _singleApprenticeship;
+        private Fixture _fixture;
+
+        public HomePageSteps(TestContext context) : base(context)
+        {
+            _fixture = new Fixture();
+            _context = context;
+            _apprenticeId = Guid.NewGuid();
+        }
+
+        [Given(@"the apprentice is authenticated")]
+        public void GivenTheApprenticeIsAuthenticated()
+        {
+            _context.Web.AuthoriseApprentice(_apprenticeId); 
+        }
+
+        [Given(@"there is a single incomplete apprenticeship")]
+        public void GivenThereIsASingleIncompleteApprenticeship()
+        {
+            _singleApprenticeship = _fixture.Build<Apprenticeship>().With(x => x.IsStopped, false)

```

Comment:
All of these setups that have to set `IsStopped` and `StoppedReceivedOn` to complimentary values suggests that I made the wrong decision to include `IsStopped` in the API.  I don't like have two separate fields that depend on being in step.

I don't think there's anything to do in this PR - maybe we can come back and remove that everywhere it's used at some point.

Reply:
An alternative is to use an extension method (be nice to have an extension property)